packages/plugin-search/src/Search/ui/LinkToDoc/index.client.tsx

```diff
@@ -47,9 +53,10 @@ export const LinkToDocClient: React.FC = () => {
           textOverflow: 'ellipsis',
         }}
       >
-        <a href={href} target="_blank">
+        <Link href={href} passHref {...{ rel: 'noopener noreferrer', target: '_blank' }}>
+          {basePath && basePath}
```

Comment:
Rendering `basePath` separate from `href` feels strange. We should just do this all with one line and remove the variable above.
```suggestion
          {`${process.env.NEXT_BASE_PATH}${href}`}
```